pkg/api/health.go

```diff
@@ -0,0 +1,25 @@
+package api
+
+import (
+	"time"
+
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/grafana/grafana/pkg/models"
+)
+
+func (hs *HTTPServer) databaseHealthyWithCache() bool {
```

Comment:
Please inline `databaseHealthyWithCache` and rename this one to drop the WithCache suffix.
```suggestion
func (hs *HTTPServer) databaseHealthy() bool {
```